modules/standard/BigInteger.chpl

```diff
@@ -557,6 +557,11 @@ module BigInteger {
     return new bigint(src);
   }
 
+  @chpldoc.nodoc
+  inline operator :(src: bool, type toType: bigint): bigint throws {
```

Comment:
I don't think we've finalized the argument names for operators (since users can't used named arguments when calling operators), but this doesn't seem to match any of the names we've typically used.

It looks like we use `t` almost universally for the type argument name.  The original value argument name varies, but the most frequent one seems to be `x`, so I'd go with that.

We may choose different names when we do finalize that decision, but again, it shouldn't be breaking

Reply:
OK, I can change those. The 2 preceding cast operators use the `src` and `toType` here, but it looks like the ones after use `x` and `t`.

Reply:
Oh, I missed there were casts above it.  I got through 100 different operator declarations and it seemed like it had enough of a pattern to draw that conclusion, at least until we finally getting around to make a general decision (where we would have to actually look at all of them).  In any case, thanks for updating them, hopefully we'll have more consistency soon